### PR TITLE
Rework 3

### PR DESCRIPTION
**Pull request by ypid**
Status: Can be reviewed.

@drybjed I failed at allowing the check_mk agent in tcpwrapper (xinetd). I did not find any examples online to configure this. Seems `hosts.allow` is checked two times, first with `xinetd` and then with `check_mk` as daemon. I tried to allow both. Did not work. Can you check that?

I have not gotten around to check #6 yet unfortunately. Not sure yet when I will come to that.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -395,4 +372,46 @@ checkmk_agent__git_dest: '{{ "/usr/local/src/check-mk/" + checkmk_agent__git_rep
...
+    by_role: 'debops-contrib.checkmk_agent'
+    rule_state: '{{ "xinetd" in checkmk_agent__type | ternary("present", "absent") }}'
+
+                                                                   # ]]]
+# .. envvar:: checkmk_agent__tcpwrappers__dependent_allow [[[
+#
+# Configuration for the debops.tcpwrappers_ Ansible role.
+checkmk_agent__tcpwrappers__dependent_allow:
+
+  - daemon: 'inetd'
```
Comment (drybjed) [inline defaults/main.yml:408]: This should be the process name, most likely either `xinetd` or `check_mk` or whatever name the process has. A similar example [can be seen in debops.nullmailer](https://github.com/debops/ansible-nullmailer/blob/master/defaults/main.yml#L338-L346) which uses `xinetd` to run the `sendmail` binary on demand.

Comment (ypid) [reply defaults/main.yml:408]: @drybjed Thanks but I tried all that. I don’t give up that easy, you know :wink:
As I said above, it seems that xinetd itself checks the particular service again. So two entries are needed.

File: defaults/main.yml [hunk 2]
```diff
@@ -379,7 +356,7 @@ checkmk_agent__plugin_nginx_servers:
 # .. envvar:: checkmk_agent__git_repo [[[
 #
 # Check_MK agent source repository.
-checkmk_agent__git_repo: 'http://git.mathias-kettner.de/check_mk.git'
+checkmk_agent__git_repo: 'https://git.mathias-kettner.de/check_mk.git'
```
Comment (ganto) [inline defaults/main.yml:359]: Nice catch. This wasn't possible when I added this feature, as the certificate was not valid for the `git` subdomain. Finally they fixed it (with Let's encrypt :smile:)

**Review by ganto — APPROVED** [review]

File: defaults/main.yml (hunk 2 above)
Comment (ypid) [reply defaults/main.yml:359]: Right, I remember, http only. Still their side deployment has some rough edges like that https://git.mathias-kettner.de redirects to http://git.mathias-kettner.de/git/.

**Comment by ganto** [comment]
Still interested in merging this? Should I have a closer look at it?

**Comment by ypid** [comment]
@ganto Yes absolutely. You can check the PR. I wanted to wait if someone has a better idea how to configure tcpwrapper because I tried it to the best of my knowledge but was not able to do this properly. But it should be good. High quality standards in DebOps :wink: Feel free to merge.

File: CHANGES.rst [hunk 3]
```diff
@@ -16,6 +16,12 @@ The current role maintainer_ is ganto_.
 
 .. _debops-contrib.checkmk_agent master: https://github.com/debops-contrib/ansible-checkmk_agent/compare/v0.1.0...master
 
+Added
+~~~~~
+
+- ``role::checkmk_agent:plugins:get`` Ansible tag to allow to skip git
+  cloning/pulling related tasks (needs to be already present to work). [ypid_]
```
Comment (ganto) [inline CHANGES.rst]: Hmn, I feel this entry is confusing. I guess this shouldn't be "allow to skip", does it?

**Comment by ganto** [comment]
Other than that it looks fine. I'm not using `tcpwrappers` so I trust you on the included changes. Thanks a lot for your effort. 👍

**Comment by ypid** [comment]
Fixed, you are welcome.

File: defaults/main.yml (hunk 1 above)
Comment (ypid) [reply defaults/main.yml:408]: > A similar example can be seen in debops.nullmailer which uses xinetd to run the sendmail binary on demand.

Thanks for the hint! Works nicely. Not sure why I was not able to figure this out before.

**Pull request merged**